package.json

```diff
@@ -30,8 +30,7 @@
     "gulp-nsp": "^0.4.5",
     "gulp-babel": "^5.1.0",
     "babel-core": "^5.5.0",
-    "nock": "*",
-    "locus": "*"
```

Comment:
Hmmm ok. It's useful for quick debugging but doesn't have to be in the package.json I guess. Will merge this in the morning, thanks!